app/femr/ui/models/admin/configure/IndexViewModelGet.java

```diff
@@ -4,21 +4,33 @@
 import java.util.Map;
 
 public class IndexViewModelGet {
-    private Map<String, Boolean> settings;
+    private Map<String, Boolean> settings;;
+    private Map<String, String> descriptions;;
 
```

Comment:
Done, pushed up the changes. Added a comment that clarifies that the first map is a name-isActive pair, and the second is a name-description pair.